messages/dashboard.json

```diff
@@ -34,5 +34,17 @@
       "start": 1722981600,
       "end": 1724191200
     }
+  },
+  {
+    "message": "Enjoying rotki? Consider ",
+    "message_highlight": "supporting us in",
```

Comment:
Why in the dashboard we are missing "the" but in welcome.json we show it?

Reply:
I remember we had a couple of cases where the didn't make sense in the highlight and this time I just reused the template.

Reply:
Can you add "the" in the new CTAs that make sense? The message highlight is the link itself?

Reply:
Yes